content/docs/hooks-reference.md

```diff
@@ -353,3 +354,31 @@ Prefer the standard `useEffect` when possible to avoid blocking visual updates.
 > Tip
 >
 > If you're migrating code from a class component, `useLayoutEffect` fires in the same phase as `componentDidMount` and `componentDidUpdate`, so if you're unsure of which effect Hook to use, it's probably the least risky.
+
+### `useDebugValue`
+
+```js
+useDebugValue(value)
+```
+
+`useDebugValue` can be used to provide a label for the React DevTools to display beside of a custom hook.
```

Comment:
"can be used to display a label for custom hooks in React DevTools"?

Reply:
I dig it